### PR TITLE
Fix constance management command without admin installed

### PR DESCRIPTION
**Pull request by syre**
Tried refactoring out `ConstanceForm` into `forms.py` and `get_values` into `utils.py`.

Fixes https://github.com/jazzband/django-constance/issues/498

### CODE REVIEW

**Comment by syre** [comment]
Can anyone take a look at this?

**Review by camilonova — APPROVED** [review]

**Comment by camilonova** [comment]
Thank you. Let's find someone else who approves so we can merge it.

**Comment by syre** [comment]
Are there any updates on this?

**Comment by syre** [comment]
@camilonova

**Comment by syre** [comment]
@Mogost @jezdez Is it possible for one of you to review as well?

File: tests/test_checks.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import datetime
 from decimal import Decimal
-import mock
 
-from constance.admin import get_values
+import mock
```
Comment (sergei-iurchenko) [inline tests/test_checks.py:4]: from unittest import mock

**Comment by sergei-iurchenko** [comment]
Merge develop, please, and fix issue. I think we can merge it

File: tests/test_checks.py (hunk 1 above)
Comment (syre) [reply tests/test_checks.py]: Done!

**Comment by syre** [comment]
@sergei-iurchenko can you have a look?

**Pull request merged**